### ISSUE TITLE
Add external link icon back into URLs in work description.

### ISSUE THREAD

**Issue by eddierubeiz** [sciencehistory/scihist_digicoll#1360]
While I was fixing [a bug that showed up in production recently](https://github.com/sciencehistory/scihist_digicoll/issues/1357) I discovered a second smaller bug, hidden by the first: the little "external link" icon we were adding to outgoing links was not displaying properly. (The first bug caused the page to crash, and thus prevented the second bug from appearing.)

I fixed the first bug, but wasn't able to figure out a timely fix for the second, so I'm adding this issue to cover the following work:
- Re-add `<i class='fa fa-external-link' aria-hidden='true'></i>&nbsp;` to the beginning of links output by `turn_bare_urls_into_links` in `app/presenters/description_display_formatter.rb ` (partially reversing [the change I made this morning](https://github.com/sciencehistory/scihist_digicoll/pull/1359/files)).
- Figure out how to get it to display properly.
- Reverse the change I made to the corresponding test.

In the meantime, the icon won't show up, which is fine for the time being.

**Comment by eddierubeiz** [comment]
Note that `auto_link` has been deprecated. We may wish to switch to https://github.com/vmg/rinku .

**Comment by jrochkind** [comment]
Did reproduce by putting an email address into the description. 

Still a VERY weird error, why would there be a method `mail_to` anywhere, or why would anything expect it? But ok, I have something to debug!

**Comment by jrochkind** [comment]
rinku does not seem to have whatever weird bug involving escaping auto_link had; good find there!

**Comment by jrochkind** [comment]
@apinkney0696  just FYI. 

OK after looking through the DB.. this is a pretty low-value feature, honestly!

What it does is turn "bare" URLs in the description into links, like if you just wrote "http://example.com" in the description (without `<a>`) tags. 

I can't find any examples of this in any of our descriptions, heh. 

It also turns bare email addresses into `mailto` links. I found only **ONE** example of this in the DB. https://digital.sciencehistory.org/works/oj07kjf

> Individuals interested in reviewing one or more of these interviews should email oralhistory@sciencehistory.org.

Which honestly probably doesn't make sense there and shouldn't be there/have ever have been there anyway!

It probably does not really matter if such a mailto link gets an "external link" icon!

At this point I've already fixed it, so it behaves as it always did (that email link WILL get an external link icon, once fixed!), but it honestly probably doens't matter. Honestly this partially shows the cost of adding features that are pretty low-value and we aren't sure we need them but we add just becuase it's easy and why not -- down the road, it accidentally crashes our app, then we spend a couple days of developer time fixing the feature, before realizing we probably don't care about the feature after all!
